### ISSUE TITLE
TextField clear

### ISSUE THREAD

**Issue by jaydeepkaria-mi** [rohanjariwala03/drop_down_search_field#46]
I need something like if user clicks outside this searchfield whilt tyoing something then if he has not selected anything from dropdown then textfield should be clear

**Comment by rohanjariwala03** [comment]
@jaydeepkaria-mi On package side I cannot determine if suggestion is selected or not in the scenario when controller was passed by you and initial value also been passed. You can wrap the package with TapRegion widget and onTapOutside you can clear the field.

```
TapRegion(
   onTapOutside: (event) {
        controller.clear();
   },
   child: DropDownSearchFormField( ...... )
)

Closing this ticket for now.
